appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectList.java

```diff
@@ -288,24 +166,25 @@ public void onClick(ClickEvent event) {
   // correct thing do to. The alternative is to add a call to the
   // ProjectManagerEventListener interface that this is the
   // implementation of.
-  public void refreshTable(boolean needToSort) {
-    if (Ode.getInstance().getCurrentView() == Ode.TRASHCAN) {
-      refreshTable(needToSort, true);
-    } else {
-      refreshTable(needToSort, false);
-    }
-  }
 
-  public void refreshTable(boolean needToSort, boolean isInTrash) {
-    if (needToSort) {
+  public void refresh(boolean needToSort) {
+    LOG.info("Refresh ProjectList");
+    List<Project> projects = folder.getProjects();
+    List<ProjectFolder> folders = folder.getChildFolders();
+//    if (needToSort) {
```

Comment:
This needs to be returned to the code base. This was an optimization to prevent O(n^2 log n) behavior on initializing the project list and taking this out will degrade performance for users with many projects in the project list.